tests/integration/vlan_test.py

```diff
@@ -79,8 +79,7 @@ def test_rollback_for_vlans(eth1_up):
     with pytest.raises(netapplier.DesiredStateIsNotCurrentError):
         netapplier.apply(desired_state)
 
-    current_state_after_apply = netinfo.show()
-    assert current_state == current_state_after_apply
+    assertlib.assert_state(current_state)
```

Comment:
If you found things failing here it is indeed interesting.
I'm pretty sure with this change it is a different assert then intended.

The rollback is supposed to revert the changes and we want to check that the new state is identical to the previous one. If we are to touch something, it is only the ordering/sorting, `assertlib.assert_state` does much more.

Reply:
OK. I will do ordering here instead of use `assertlib.assert_state()`.